streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -2895,10 +2871,8 @@ public void shouldNotCommitOnHandleAssignmentIfNoTaskClosed() {
         expect(activeTaskCreator.createTasks(anyObject(), eq(Collections.emptyMap()))).andReturn(Collections.emptySet());
         expect(standbyTaskCreator.createTasks(eq(assignmentStandby))).andReturn(singletonList(task10));
         expect(standbyTaskCreator.createTasks(eq(Collections.emptyMap()))).andReturn(Collections.emptySet());
-        topologyBuilder.addSubscribedTopicsFromAssignment(eq(asList(t1p0)), anyString());
```

Comment:
Is the idea here that since we do not rely on the return value from this method you do not want to verify this interaction? I noticed that earlier in the pull request we do verify interactions of this type.

Reply:
The idea is to have a unit test that specifically verifies this interaction, i.e., `shouldAddSubscribedTopicsFromAssignmentToTopologyMetadata()` and to not verify this interaction in the other unit tests where we verify other unrelated aspects. I think this interaction was verified in so many unit tests before because the topology builder mock was specified as strict and easy mock requires to verify all interactions with strict mocks even if we actually do not care about this interactions in some unit tests.

Reply:
Got it, okay, this makes a lot of sense.